relay-profiling/src/android.rs

```diff
@@ -160,8 +175,30 @@ fn parse_profile(payload: &[u8]) -> Result<AndroidProfile, ProfileError> {
     Ok(profile)
 }
 
-pub fn parse_android_profile(payload: &[u8]) -> Result<Vec<u8>, ProfileError> {
-    let profile = parse_profile(payload)?;
+pub fn parse_android_profile(
+    payload: &[u8],
+    tags: BTreeMap<String, String>,
+) -> Result<Vec<u8>, ProfileError> {
+    let mut profile = parse_profile(payload)?;
+
+    if let Some(transaction_name) = tags.get("transaction") {
+        profile.transaction_name = transaction_name.to_owned();
+
+        if let Some(ref mut transaction) = profile.transaction {
+            transaction.name = profile.transaction_name.to_owned();
+        }
+    }
+
+    if let Some(release) = tags.get("release") {
+        profile.release = release.to_owned();
+    }
+
+    if let Some(environment) = tags.get("environment") {
+        profile.environment = environment.to_owned();
+    }
+
+    profile.tags = tags;
```

Comment:
Out of curiosity: Why do we store release, transaction name and environment in _both_ tags _and_ dedicated fields? I.e., why not remove them from `tags`?

Reply:
No real reason, it's mostly because it started as collecting tags and evolved into something else. I'll remove them.